src/App.tsx

```diff
@@ -26,10 +27,16 @@ function App() {
           <Routes>
             <Route path={PATHS.INDEX} element={<MainPage />} />
             <Route path={PATHS.PRODUCT} element={<ProductPage />} />
-            <Route path={PATHS.CART} element={<CartPage />} />
+            <Route
+              path={PATHS.CART}
+              element={<AuthPage element={<CartPage />} />}
+            />
             <Route path={PATHS.SIGNIN} element={<SigninPage />} />
             <Route path={PATHS.SIGNUP} element={<SignupPage />} />
-            <Route path={PATHS.PROFILE} element={<ProfilePage />} />
+            <Route
+              path={PATHS.CART}
+              element={<AuthPage element={<ProfilePage />} />}
+            />
```

Comment:
페이지 컴포넌트라기 보다는, 로그인 상태를 체크해주는 컴포넌트인 것 같군요. 
PrivateRoute, AuthRoute 등 으로 검색해보셔서 아래와 같이 변경해보는 건 어떨까요?

```suggestion
            <PrivateRoute
              path={PATHS.CART}
              element={<ProfilePage />}
            />
```

Reply:
좋은 의견 감사합니다.😁
PrivateRoute를 찾아보니, PublicRoute을 사용하는 예시도 봤네요.
단순히 Route를 사용하는 것이 아닌, Route를 하는 상황에 따라서 한번 더 Layer로 감싸서 추상화한다는 것이 좋은 방식인 것 같네요.
앞으로의 프로젝트에서 녹여보도록 하겠습니다.